build.sbt

```diff
@@ -20,6 +20,11 @@ lazy val databaseMigration = (project in file("database/migration"))
   .dependsOn(core)
   .withTestSettings
 
+lazy val dockerHashing = (project in file("dockerHashing"))
+  .settings(dockerHashingSettings: _*)
+  .dependsOn(core)
```

Comment:
OK let's at least wait until the dust settles on all the work in this area before trying to do this.  Though given the reusable trait it sounds like these subprojects are only growing more entangled.  🙂